UnitsNet.Tests/GeneratedCode/IQuantityTests.g.cs

```diff
@@ -252,111 +252,111 @@ public void QuantityInfo_IsSameAsStaticInfoProperty()
         [Fact]
         public void Type_EqualsStaticQuantityTypeProperty()
         {
-            void Assertion(QuantityType expected, IQuantity quantity) => Assert.Equal(expected, quantity.Type);
```

Comment:
I believe we should duplicate the assertions to cover both old and new functionality, until removed. Here and other places.